src/app/services/notification.service.ts

```diff
@@ -23,6 +23,7 @@ import { Subject } from 'rxjs';
 export class NotificationService {
   public notification$: Subject<string> = new Subject<string>();
   public error$: Subject<string> = new Subject<string>();
+  public importantNotification$: Subject<string> = new Subject<string>();
```

Comment:
This seems to be unused?

Reply:
It is, I forgot to update after changing the app.component.ts. Fixed now.